spark/BidResponseJob.scala

```diff
@@ -0,0 +1,191 @@
+package com.liveintent.dwh.logs.jobs
+
+import com.liveintent.dwh.dmp.models._
+import com.liveintent.dwh.dmp.sources.BidderGroupInfoSource
+import com.liveintent.dwh.spark.util.{SparkArgs, SparkDwhJob}
+import com.liveintent.dwh.userver.sources.ImpressionSource
+import com.liveintent.util.logparser.LogParser
+import com.twitter.algebird.Moments
+import org.apache.spark.rdd.RDD
+
+
+class BidResponsesJob(override val args: SparkArgs) extends SparkDwhJob(args) {
+
+  import BidResponsesJob._
+
+  private lazy val impressions = ImpressionSource()
+    .map{ _.sanitizeToPositive }
+    .filter { i => i.isFirstImpression && i.demandType == Some("default") }
+    .groupBy { e => (e.decisionId.getOrElse(""), e.redecisionAttempt.getOrElse(-1)) }
+    .mapValues(_.take(1))
+    .map( _._2 )
+
+  private lazy val brokenDropTriplets = impressions.filter( _.isBrokenDrop )
```

Comment:
Maybe add a comment on what a broken drop is and why we need to remove them manually below (rather than just using the isBrokenDrop flag on each row)